waspc/src/Wasp/Analyzer/Parser.hs

```diff
@@ -19,7 +19,8 @@ module Wasp.Analyzer.Parser
     WithCtx (..),
     withCtx,
     ctxFromPos,
-    getCtxPos,
+    ctxFromRgn,
+    getCtxRgn,
```

Comment:
I see there is no love for vowels here :D

Reply:
Haha, to abyss with them! Yeah younger me would object, but now I started doing this because I am calling these on lot of places and smaller names make them not stand out so much. But it is somewhat questionable.